<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createOrder</class>
 <widget class="QDialog" name="createOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create/Update Order</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item row="3" column="0">
    <widget class="QPushButton" name="addProdBtn">
     <property name="text">
      <string>Add products</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="dateLb">
     <property name="text">
      <string>Date of order:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="choseUserBtn">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Select user</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="choseWorkerBtn">
     <property name="text">
      <string>Select worker</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="prodCountLb">
     <property name="text">
      <string>Count of products:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="prodCountEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cencalBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="workerIDEdit">
     <property name="toolTip">
      <string>Enter worker ID</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QDateEdit" name="dateEdit">
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="userIDEdit">
     <property name="toolTip">
      <string>Enter user ID</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>choseUserBtn</tabstop>
  <tabstop>userIDEdit</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>choseWorkerBtn</tabstop>
  <tabstop>workerIDEdit</tabstop>
  <tabstop>addProdBtn</tabstop>
  <tabstop>prodCountEdit</tabstop>
  <tabstop>okBtn</tabstop>
  <tabstop>cencalBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
